uuid.js

```diff
@@ -14,7 +14,7 @@
   // Node.js crypto-based RNG - http://nodejs.org/docs/v0.6.2/api/crypto.html
   //
   // Moderately fast, high quality
-  if (typeof(_global.require) == 'function') {
+  if (typeof(_global.require) == 'function' && typeof(module) != 'undefined' && module.exports) {
```

Comment:
Test it outside of node REPL and you'll see a different picture

   (function() {
     var _global = this;
     console.log(_global.require);
   }).call(this);